packages/medusa/src/migrations/1698056997411-product-sales-channels-link.ts

```diff
@@ -0,0 +1,38 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import IsolateProductDomain from "../loaders/feature-flags/isolate-product-domain"
+
+export const featureFlag = IsolateProductDomain.key
```

Comment:
question: Should we add a dedicated Sales Channel feature flag?

Reply:
I think we can, especially that we will have followup pr's

Reply:
good suggestion, I will add